Car1/src/car1/CarSetup.java

```diff
@@ -0,0 +1,104 @@
+
+package car1;
+
+/**
+ *The CarSetup class contains methods that will be used in the Car1 program.
+ * @author mkavic
+ */
+public class CarSetup {
+   
+   // 
+   private int yearModel; // car's year model
+   private String make; // car's makes
+   private int speed = 0; //car's speed.
+  
+   /**
+    * Constructor
+    * @param yearMod car's year model
+    * @param mak car's make
+    * @param sped car's speed
+    */
+   public CarSetup(int yearMod, String mak, int sped)
+   {
+       yearModel =  yearMod;
+       make = mak;
+       speed = sped;
```

Comment:
Check the assignment I believe it would like you to set speed 0 in the constructor and not in its field.